rebasehelper/checkers/licensecheck_tool.py

```diff
@@ -72,7 +72,7 @@ def get_license_changes(cls, old_dir, new_dir):
                 if 'GENERATED FILE' in dlicense:
                     continue
                 file_path = os.path.relpath(file_path, source_dir)
-                if re.search(r'COPYING|LICENSE', file_path):
+                if re.search(r'COPYING|LICENSE', file_path, re.IGNORECASE):
```

Comment:
This covers roughly 63% of all variants used in Fedora specs, if you add also `COPYRIGHT`, `LICENCE` and `LICENSING`, it will cover another 10%, which is sufficient for our purposes I think.
There is a number of license files named after the license, so adding `GPL` or `BSD` could be also considered.